src/main/java/seedu/wife/commons/core/HelpMenu.java

```diff
@@ -0,0 +1,53 @@
+package seedu.wife.commons.core;
+
+/**
+ * Menu of commands to be displayed in Help Window.
+ */
+public enum HelpMenu {
+    ADD("Add food item - ", "add n/NAME u/UNIT q/QUANTITY e/EXPIRY DATE [t/TAG]"),
+    UPDATE("Update food item - ", "update <Old Item> /to <New Item>"),
+    DELETE("Delete food item - ", "delete <Index>"),
+    TAG("Tag food item - ", "tag <Index> /with <Tag Name>"),
+    LIST("List food items - ", "list"),
+    EXIT("Exit WIFE - ", "exit");
```

Comment:
I like how you used enums but there are some inconsistencies in representing the command format.

Also, for delete command, the format has been changed to `del <Index>`